src/consensus/params.h

```diff
@@ -97,11 +97,27 @@ struct LLMQParams {
     // should at the same time not be too large so that not too much space is wasted with null commitments in case a DKG
     // session failed.
     int dkgMiningWindowEnd;
+
+    // Each member will sleep for a random time between 0 and dkgRndSleepTime milliseconds. The purpose of this is to
+    // avoid overloading the network due to all members sending out expensive-to-verify DKG messages at once
     int dkgRndSleepTime;
```

Comment:
Why is this a part of consensus? Why random? Can't each member sleep `idx * dkgSleepTimeoutPerMemeber` instead?

Reply:
Hmm, agree, this should not have been in consensus. I'll move this to chain params.

Reply:
Removed from consensus and made sleep time deterministic